### PR TITLE
cheeky breeky ;)

### PR DESCRIPTION
**Pull request by baarzelim**
its Tsuk

### CODE REVIEW

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+     isPerson: false
+     isOlder: true
+   - name: nadav
+     age: as old as the universe
+     isPerson: true
+     isOlder: true
+# object / map
+family:
+      sisters:
+       gaya:
```
Comment (noamu111) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: fix the indentation

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+         kids: null
+         age: 18
+         isOlder: null
+         dateOfBirth: 9.10.2002
+       noga:
+        kids: null
```
Comment (noamu111) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: bad indetation

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        age: 14 but acts like a 2 yr old
+        isOlder: false
+        dateOfBirth: 18.4.2007
+       hila:
+         kids: null
+         age: 14 but acts like a 3 yr old
+         isOlder: false
+         dateOfBirth: 18.4.2007
+      parents:
+       ziv:
```
Comment (noamu111) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: you are very bad at indentation

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+name: Tsuk
+# number
+age: 18.5
+isPerson: true
+# array
+food: [pizza,hamburger, steak]
+# array option 2
+friends:
+   - name: darkness
+     age: old
```
Comment (omershilohpsi) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml:11]: age type is integer

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+     isPerson: false
+     isOlder: true
+   - name: nadav
+     age: as old as the universe
```
Comment (omershilohpsi) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml:15]: age type is integer

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+family:
+      sisters:
+       gaya:
+         kids: null
+         age: 18
+         isOlder: null
+         dateOfBirth: 9.10.2002
+       noga:
+        kids: null
+        age: 14 but acts like a 2 yr old
```
Comment (omershilohpsi) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: age is integer

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+        isOlder: false
+        dateOfBirth: 18.4.2007
+       hila:
+         kids: null
+         age: 14 but acts like a 3 yr old
```
Comment (omershilohpsi) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: again age is integer

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+     isOlder: true
+   - name: nadav
+     age: as old as the universe
+     isPerson: true
+     isOlder: true
+# object / map
+family:
+      sisters:
+       gaya:
+         kids: null
```
Comment (omershilohpsi) [inline yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: a null value can be unwritten

**Review by omershilohpsi — CHANGES_REQUESTED** [review]
lech tetaken et hakol

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml (hunk 3 above)
Comment (etayruben) [reply yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: hahahaha

Comment (nadav7679) [reply yaml/Tsuk/yaml_exercise1_Tsuk.yaml]: lol

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml (hunk 4 above)
Comment (nadav7679) [reply yaml/Tsuk/yaml_exercise1_Tsuk.yaml:11]: shut up Omer lol

File: yaml/Tsuk/yaml_exercise1_Tsuk.yaml (hunk 5 above)
Comment (nadav7679) [reply yaml/Tsuk/yaml_exercise1_Tsuk.yaml:15]: Omer is right.

**Review by omershilohpsi — APPROVED** [review]

**Review by benedery — APPROVED** [review]

**Pull request merged**